src/app/external-cluster-wizard/steps/external-cluster/provider/eks/template.html

```diff
@@ -110,29 +111,48 @@
                [formControlName]="Controls.Vpc">
     </km-select>
 
-    <mat-form-field *ngIf="isDialogView()">
-      <mat-label>VPC</mat-label>
-      <input matInput
-             type="text"
-             autocomplete="off"
-             [formControlName]="Controls.Vpc" />
-    </mat-form-field>
 
-    <km-chip-autocomplete label="Subnet"
-                          placeholder="Select Subnet IDs"
-                          [tags]="subnetIds"
-                          [required]="true"
-                          [disabled]="form.get(Controls.SubnetIds).disabled"
-                          [formControlName]="Controls.SubnetIds"></km-chip-autocomplete>
-
-    <km-chip-autocomplete *ngIf="!isDialogView()"
-                          label="Security Groups"
-                          placeholder="Select SecurityGroup IDs"
-                          [tags]="securityGroupIds"
-                          [required]="true"
-                          [disabled]="form.get(Controls.SecurityGroupsIds).disabled"
-                          [formControlName]="Controls.SecurityGroupsIds">
-    </km-chip-autocomplete>
+    <div *ngIf="isDialogView()">
+      <mat-form-field>
+        <mat-label>VPC</mat-label>
+        <input matInput
+               type="text"
+               autocomplete="off"
+               [formControlName]="Controls.Vpc" />
+        <mat-hint>Selected VPC for your nodes.</mat-hint>
```

Comment:
```suggestion
        <mat-hint>Select VPC for your nodes.</mat-hint>
```

Reply:
in the dialog view (add MD) the VPC is the same that been use for the cluster so it's disable so the user can't `select` it's already `selected`

Reply:
yes! i know but it was looked very add due to spacing + it there should be info as hint therefore i added. but yes i can update from `select` to `selected`